### ISSUE TITLE
Add jar in maven repo

### ISSUE THREAD

**Issue by dukke** [dukke/FXRibbon#9]

**Comment by dukke** [comment]
@Biggo6 @klolo It's now on Bintray, check instructions at www.pixelduke.com/fxribbon 
 Does this work for you?

**Comment by Biggo6** [comment]
@dukke thanks this looking amazing and helping

**Comment by klolo** [comment]
![obraz](https://user-images.githubusercontent.com/18196270/38636110-9f86456e-3dc7-11e8-90a4-5e9fc67531f9.png)
Yes, but items are align to right side. How can I put it to the left?

My gradle configuration for this dependency:
```
repositories {
    mavenCentral()
    maven { url "https://dl.bintray.com/dukke/maven" }
}
dependencies {
  compile 'com.pixelduke:FXRibbon:1.0.0'
```

**Comment by dukke** [comment]
Version 1.0.1 should fix this.. tell me if it does

**Comment by dukke** [comment]
@klolo did the problem get fixed in the latest version (1.0.1)?
